clang/lib/CodeGen/CGHLSLRuntime.h

```diff
@@ -104,6 +104,8 @@ class CGHLSLRuntime {
   GENERATE_HLSL_INTRINSIC_FUNCTION(SClamp, sclamp)
   GENERATE_HLSL_INTRINSIC_FUNCTION(UClamp, uclamp)
 
+  GENERATE_HLSL_INTRINSIC_FUNCTION(CreateResourceGetPointer,
```

Comment:
Why "CreateResourceGetPointer"? Shouldn't this just be "ResourceGetPointer"?

Reply:
I was following the example of `CreateHandleFromBinding` below. I have no problem changing it if you want.